Source/CBLInternal.h

```diff
@@ -160,4 +161,9 @@ NSString* CBLKeyPathForQueryRow(NSString* keyPath); // for testing
 - (instancetype) initWithDatabase: (CBLDatabase*)database
                            remote: (NSURL*)remote
                              pull: (BOOL)pull                       __attribute__((nonnull));
+@end
+
+
+@interface CBLCookieStorage ()
+- (void) reset;
```

Comment:
Why does this method need to be in a separate header? It seems like part of the class's regular API.